SCAutolib/models/file.py

```diff
@@ -261,13 +348,22 @@ def save(self):
                 # to empty parser object
                 self._changes = ConfigParser()
                 self._changes.optionxform = str
+        os.chmod(self._conf_file, 0o600)
 
     def clean(self):
```

Comment:
If it is desired to use this specific implementation, it should be renamed to "restore".